src/tutorials.md

```diff
@@ -0,0 +1,23 @@
+# Tutorials
+
+In this section, we will walk you through some examples that use the Alan language:
+
+  - [Getting Started](./getting_started.md)
+  - [The many forms of "Hello, World!"](./many_forms_hello_world.md)
+  - [More advanced examples](./advanced_examples.md)
+
+<!---
+TODO proposal
+
+In this section, we will go through a series of steps to go from zero a Web server deployed in AWS that handles restaurant orders and saves them.
+
+  - Install Alan
+  - Compile and run an Alan project
+  - Many Forms of Hello World
```

Comment:
I'd probably merge "Many forms of Hello World" with "More advanced examples" and drop several of the minor variants on "Hello World" in the process. I was using that back in the Java interpreter days as a kind of test suite before we had the BDD suite, so lots of tiny variations are in there.

Reply:
Ah yes, that also doesn't really belong in the end version of the tutorial since is not related to the web server

Reply:
Actually the "Many forms of Hello World" reads well as a topic under "About Alan"!